ui/app/styles/components/tabs.scss

```diff
@@ -45,11 +47,11 @@
 
   a,
   .tab {
```

Comment:
I have a marketplace command that alphabetizes the css within blocks. You can see that impacting this block here, where the only changes are alphabetizing.